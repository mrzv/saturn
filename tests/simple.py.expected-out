╔══════════════════════════════════════════════════════════════════════════════╗
║                            Sample Saturn Notebook                            ║
╚══════════════════════════════════════════════════════════════════════════════╝

Some test markup:                                                               

 • item 1                                                                       
 • item 2                                                                       

First, we import some necessary boilerplate.                                    

>>> import sys, os                                                              
... import numpy as np                                                          


Fix seed for testing                                                            

>>> np.random.seed(42)                                                          


Set up the matrix a on which we operate.                                        

>>> a = np.random.random((10,20))                                               


We can examine a's shape:                                                       

>>> print(a.shape)                                                              

(10, 20)

>>> a.shape                                                                     

(10, 20)

>>> ## Is this the best demarkation of the evaluated expression?                
...                                                                             
... b = 5                                                                       


>>> print(b)                                                                    

5

>>> import matplotlib.pyplot as plt                                             
... plt.matshow(a)                                                              
... plt.show()                                                                  


>>> # vim: ft=python foldmethod=marker foldlevel=0                              


