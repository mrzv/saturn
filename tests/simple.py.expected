#m> # Sample Saturn Notebook
#m>
#m> Some **test** *markup*:
#m>
#m>  - item 1
#m>  - item 2
#m>
#m> First, we import some necessary boilerplate.

import sys, os
import numpy as np

#m> Fix seed for testing

np.random.seed(42)

#m> Set up the matrix `a` on which we operate.

a = np.random.random((10,20))

#m> We can examine `a`'s shape:

print(a.shape)

#o> (10, 20)

a.shape

#o> (10, 20)

#chk>{{{
#chk>gANDIL74p48pahlbl9cc9HhvEdzvygk3xLnyHUyg9cZOkYQgcQAugAN9cQAoWAwAAABfX2J1aWx0aW5z
#chk>X19xAWNidWlsdGlucwpfX2RpY3RfXwpYAwAAAHN5c3ECY2RpbGwuX2RpbGwKX2ltcG9ydF9tb2R1bGUK
#chk>cQNYAwAAAHN5c3EEhXEFUnEGWAIAAABvc3EHaANoB4VxCFJxCVgCAAAAbnBxCmgDWAUAAABudW1weXEL
#chk>hXEMUnENWAEAAABhcQ5jZGlsbC5fZGlsbApfZ2V0X2F0dHIKcQ9oA1gcAAAAbnVtcHkuY29yZS5fbXVs
#chk>dGlhcnJheV91bWF0aHEQhXERUnESWAwAAABfcmVjb25zdHJ1Y3RxE4ZxFFJxFWNudW1weQpuZGFycmF5
#chk>CnEWSwCFcRdDAWJxGIdxGVJxGihLAUsKSxSGcRtjbnVtcHkKZHR5cGUKcRxYAgAAAGY4cR1LAEsBh3Ee
#chk>UnEfKEsDWAEAAAA8cSBOTk5K/////0r/////SwB0cSFiiUJABgAA7FFfHnf41z9U1rtoQGzuP1FnHo9+
#chk>bOc/sCxj1jUo4z+EISI3a/jDP0x/meGg98M/cFVknSK9rT+1t1UJt7frP61LkoxVPOM/goKa0oeo5j+A
#chk>fjgHGhSVP6VT42KACe8/9hPBv16j6j+YMAOQ7S3LPzgZ4F8KRsc/SDThicx5xz9oquR0tHjTP2M8BwDO
#chk>yuA/WjsvuPyk2z8OvTKMf6PSPyd1mYVMlOM/fCS/UO/awT9UQMp3f7LSPzi33PF4ctc/3I+5KEAw3T8u
#chk>OnlWKSDpP3SQHhbpjsk/E9Tux5t04D8X991lD/XiP3BDCTdZyKc/iqK45gFx4z+0PjUHvNPFP6CqbKA4
#chk>p7A/idDEM0Vd7j+gzkAmdebuP5+qih1k3uk/6rIgwMp+0z9we2MnCgG5P2HP9qg85eU/3FJ3XXUr3D+w
#chk>F4XT5T2/Pxhiqn76sN8/AGgYS1+boT8xcpAZJxntPzA4denZj9A/dn0F72Ez5T8euGYDE/PTP6ju1LBl
#chk>pOA/PFNFjqZ+4T9cQoCQT6nHP+Z/XlfWBu8/WIzMWePN6D9DopQVYBDuPw/mtPdsouw//bnoIv8h4z+0
#chk>YE5l/n/tP7gTQdJxp7Y/TFZVaPcVyT+g14P9ByinP/aawU420tQ/9oL6tRbg2D8aSlFUyF3RP0zVWIYE
#chk>heo/xte45wvV1j/ENNu81PrRP4efGi3EXeE/SIm9FM4Jwj+8O6kAmavpP+gXgEDAFbM/KJCx6ZOU7z/S
#chk>W5SpOrboP6ggsvSDb8k/AHg1B1yedj9XncuQQhjqPwZq2EqTnuY/xiMp1wZU5z/FgGwmP67oP7hBpB2X
#chk>9LI/3mc1PRrx1j+AWUc9mKm9P4NwZhOLnus/zDzM6Q7y4z9gZsPobi3VPzi+3StcRbA/cMlZZSLn0z8Y
#chk>z1O1zc/UPyp9XQ7vWOc/q1AZ7d5m5D+Ytj76C2TsP/Y7EPPEON4/8EMYf7qdvj+qSWW75tLmPxAN4+JZ
#chk>WOg/B6bxmPv14T/GNgZdw6voP9jOUthYmt8/A7jXMjq64D+6GuMA1VzbP6AtPLh4B5o/0Ht0xsWeuz/Q
#chk>MFR5fBegP3PH5l15XeQ/npaBjGge1D9eNcgKNkbgP+esjtjICu0/jLGryc7ozz80mFG8tkPaP0jvz5R5
#chk>Leg/oO0VH0JJzT/o5ymT9LSzP2brjq1Ji9I/cGUWLuaixD+qVn9KFcDtP5i6LkUf3Ok/rwZs9NdE5D/M
#chk>fdNRAePrP9mu34Cut+k/VAaqFofhxz+eeojj14/sPxkhTKlKQuE/nwR9vIzW6T9X2Wypx6zsP0QvQT8r
#chk>WtQ/sE616FwsvD9Q8Fy6+izNP9IQAei7Vds/zwdpTS0t6j+UGDjdGovrP4DFTVTWeXw/VugiugpY4D+I
#chk>7uCj3LbaP8w3FFsHbsw/mAafKH+vvj8U+POpfJvVP3xUrPhQLO4/dAfAWVuv1D8XfzvK7pngPzqHnZ0h
#chk>f+Y/OOaEGLVF1z9mAgS91hjvPxTN/ERezO4/+AeSfTMd0D/surJl69LfPzJILxmXQdM/XF/7n9M60j+g
#chk>8BKR2OKiP8AN6A+NgeM/4+akUfIV4D/QgB1CbFuqP7a0wvpX1dE/ZpOJnoMQ7T9kK+TK9qnOP0C1fkjq
#chk>i8I/Vg+VqzFT3z+bKTbScorvP4yve8mq+84/NstSaCKC5T+yhnoZMF/oPxjj3zPoas4/RGo2X4xN5z/6
#chk>LMNDwonXP3kG72/ZO+Q/b2WYGeBF5D+3BkTzECXhPxALjPk6Hbc/s1WaTMy66j+EyhAcqYfUP5wMIqvW
#chk>38c/wFnvtnrgpD8ryE1RmOjiP96G4XSbruU/oPIGW2b8kD/X7Tz7EGPgP8gilEXQ/cw/G3xiaEGl5D8c
#chk>Ygyfo1HGPx4dl3UpHOY/0FoknEXA2D+t0FkrsfntP3Cgz0pJmsE//Ljh9QfU1T/wtrDGmQy9P1ZxKhIX
#chk>l+0/tHvI+ikT7D8IEtSZHYLQP2Ins9yWHuU/ib3tK68m6j//CBF+NMThP80WC8Xl8uA/uKfwDwT1zj8Y
#chk>t+Q+ldW3P0exO9L9tew/0tGHhznQ7D9vPMH8XULkP+YgOwKqstU/NFWaHXNZ1j8j+2JnBzvnP0bgbJMg
#chk>tew/YDeeEQNj7D+aE42PvfToP3EidHEjYnUu
#chk>}}}

## Is this the best demarkation of the evaluated expression?

b = 5

#---#

print(b)

#o> 5

import matplotlib.pyplot as plt
plt.matshow(a)
plt.show()

#o> png{{{
#o> pngiVBORw0KGgoAAAANSUhEUgAAA8AAAAHgCAYAAABq5QSEAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAP
#o> pngYQAAD2EBqD+naQAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMi4yLCBodHRwOi8v
#o> pngbWF0cGxvdGxpYi5vcmcvhp/UCwAAHmlJREFUeJzt3HuQlfWd5/FvX6RB07RXbksLqKhR8RJRxktSOlIi
#o> pngq4zOpjLqmhQhickavDLGyGxELaPETMYhJhZGq4y463V2Ss3GDZkUI7puvILROJXxStZGAyjRbsDYQPfZ
#o> pngP7bsSdfTGkdofge/r1fVKYtDH+tT9MPDeffzdDfUarVaAAAAwMdcY+kBAAAAsC0IYAAAAFIQwAAAAKQg
#o> pnggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABIQQADAACQ
#o> pngggAGAAAgBQEMAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAA
#o> pngQAoCGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAA
#o> pngAAApCGAAAABSEMAAAACkIIABAABIQQADAACQggAGAAAgBQEMAABACgIYAACAFAQwAAAAKQhgirvhhhti
#o> png/PjxMXTo0JgyZUo88cQTpSfBVnPFFVdEQ0NDv8f+++9fehZskYcffjhmzJgRY8aMiYaGhrjvvvv6/X6t
#o> pngVot58+bF6NGjY9iwYTF16tR48cUXC62Fj+ZPHedf/OIXK+f3k046qdBa4MMSwBR19913x5w5c+Lyyy+P
#o> png5cuXxyGHHBLTpk2LNWvWlJ4GW82BBx4Yv/vd7/oejzzySOlJsEU2bNgQhxxySNxwww0D/v53v/vduP76
#o> png6+PGG2+Mxx9/PHbaaaeYNm1avPvuu9t4KXx0f+o4j4g46aST+p3f77zzzm24EPgomksPILfrrrsuzj77
#o> png7Jg1a1ZERNx4443xwAMPxC233BKXXnpp4XWwdTQ3N8eoUaNKz4CtZvr06TF9+vQBf69Wq8WCBQviW9/6
#o> pngVpx66qkREXHbbbfFyJEj47777oszzjhjW06Fj+yDjvP3tLS0OL/DdsYVYIrZuHFjLFu2LKZOndr3XGNj
#o> pngY0ydOjUeffTRgstg63rxxRdjzJgxsddee8VZZ50Vr776aulJMGhWrFgRq1at6ndub2triylTpji387Gz
#o> pngdOnSGDFiROy3335xzjnnxNq1a0tPAv4EAUwxb775ZvT09MTIkSP7PT9y5MhYtWpVoVWwdU2ZMiVuvfXW
#o> pngWLx4cSxcuDBWrFgRn/70p2PdunWlp8GgeO/87dzOx91JJ50Ut912WyxZsiSuvfbaeOihh2L69OnR09NT
#o> pngehrwAdwCDTCI/vj2uYMPPjimTJkS48aNi3vuuSe+/OUvF1wGwJb449v5J02aFAcffHDsvffesXTp0jjh
#o> pnghBMKLgM+iCvAFLP77rtHU1NTrF69ut/zq1ev9v00fGztvPPOse+++8ZLL71UegoMivfO387tZLPXXnvF
#o> png7rvv7vwOdU4AU8yQIUPi8MMPjyVLlvQ919vbG0uWLImjjjqq4DIYPOvXr4+XX345Ro8eXXoKDIoJEybE
#o> pngqFGj+p3bu7q64vHHH3du52Nt5cqVsXbtWud3qHNugaaoOXPmxMyZM2Py5Mlx5JFHxoIFC2LDhg19PxUa
#o> pngtncXX3xxzJgxI8aNGxevv/56XH755dHU1BRnnnlm6Wnwka1fv77fVa4VK1bEr371q9h1111jzz33jAsv
#o> pngvDC+/e1vx8SJE2PChAlx2WWXxZgxY+K0004ruBr+fT7oON91113jyiuvjM9+9rMxatSoePnll+OSSy6J
#o> pngffbZJ6ZNm1ZwNfCnCGCKOv300+ONN96IefPmxapVq+LQQw+NxYsXV354CmyvVq5cGWeeeWasXbs29thj
#o> pngjzj22GPjscceiz322KP0NPjInnrqqTj++OP7fj1nzpyIiJg5c2bceuutcckll8SGDRviq1/9arz99ttx
#o> png7LHHxuLFi2Po0KGlJsO/2wcd5wsXLoxnn302Fi1aFG+//XaMGTMmTjzxxLjqqquipaWl1GTgQ2io1Wq1
#o> png0iMAAABgsPkeYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDB1obu7O664
#o> png4oro7u4uPQUGjeOcDBznZOA4h+1XQ61Wq5UeAV1dXdHW1hadnZ0xfPjw0nNgUDjOycBxTgaOc9h+uQIM
#o> pngAABACgIYAACAFJpLD2D71dvbG6+//nq0trZGQ0PDFv2/urq6+v0XPo4c52TgOCcDxzn1qlarxbp162LM
#o> pngmDHR2Oha50B8DzAf2cqVK6O9vb30DAAA4I90dHTE2LFjS8+oS64A85G1trZGRMS+Z8+LpiFDC6/5N/ef
#o> png+/3SEyouXXlS6QkDerqj/k6M43/YU3rCdqHxt6tKT6j44pJlpScMaN6vZpSeUDFx1JulJ1RsmrG29ISK
#o> png//7c46UnVNy5blzpCQO695KppSdUNP+h/s7ntS27YW1QrBs/rPSEijeOqL/PXUREy5r6S5e2V3pLT+in
#o> pngZ9O78cy93+57n05V/R1FbDfeu+25acjQaGqpnwBuba2/2z122GlI6QkDatyxfj5v72luqs9/dOtNY2P9
#o> pngHVM7tjaVnjCgejzO6/GcUGvYofSEiuF1eD4fVqvPt07NzfV3nDc319/5vB4DuJ4uIryncVj9fe4iIpqG
#o> png1t/fv6Yh9RXA79nSb0/8OKu/f1kAAABgEAhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAA
#o> pngQAoCGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggJO74YYbYvz48TF06NCYMmVKPPHEE6UnAQAADAoBnNjd
#o> pngd98dc+bMicsvvzyWL18ehxxySEybNi3WrFlTehoAAMBWJ4ATu+666+Lss8+OWbNmxQEHHBA33nhj7Ljj
#o> pngjnHLLbeUngYAALDVCeCkNm7cGMuWLYupU6f2PdfY2BhTp06NRx99dMDXdHd3R1dXV78HAADA9kIAJ/Xm
#o> pngm29GT09PjBw5st/zI0eOjFWrVg34mvnz50dbW1vfo729fVtMBQAA2CoEMB/a3Llzo7Ozs+/R0dFRehIA
#o> pngAMCH1lx6AGXsvvvu0dTUFKtXr+73/OrVq2PUqFEDvqalpSVaWlq2xTwAAICtzhXgpIYMGRKHH354LFmy
#o> pngpO+53t7eWLJkSRx11FEFlwEAAAwOV4ATmzNnTsycOTMmT54cRx55ZCxYsCA2bNgQs2bNKj0NAABgqxPA
#o> pngiZ1++unxxhtvxLx582LVqlVx6KGHxuLFiys/GAsAAODjQAAnd+6558a5555begYAAMCg8z3AAAAApCCA
#o> pngAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCC
#o> pngAAYAACCF5tID2P5tnLIumnbcVHpGn5knf7n0hIq3Ju1cesKAhu1ef18DW/GXtdITKj77H/9P6QkVT8yZ
#o> pngXHpCxZzFE0tPGFDTbt2lJ1RsnvFO6QkVv73jk6UnVJz2uQNLT6hofun10hMGdNfy60tPqNixoan0hIob
#o> png3ppUekLFP35vaukJFQf87erSEwb07s319x7htcaxpSf00/Nu/b23qzf+hAAAAEhBAAMAAJCCAAYAACAF
#o> pngAQwAAEAKAhgAAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABIQQADAACQggAGAAAgBQEMAABACgIYAACA
#o> pngFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQEMAAAACkIYAAA
#o> pngAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAAAAApCGAAAABSEMAAAACkIIAB
#o> pngAABIQQADAACQggAGAAAgBQEMAABACs2lB7D9G391dzQ3lV7xb54/e7fSEyr2/dazpScM6IZ/WVp6QsUF
#o> pngF59XekLFg5+aWHpCRc+4ltITqhp7Sy8Y0OYNO5SeULHp4L1KT6hYfvRNpSdUXLb3n5WeUHHBHg+XnjCg
#o> pngf930idITKr7y+MzSEyruP2ph6QkVD//zfqUnVHTvPaL0hAG1/KeXSk+o+MRf/YfSE/rp2VgrPaHuuQIM
#o> pngAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQE
#o> pngMAAAACkIYAAAAFIQwAAAAKQggBObP39+HHHEEdHa2hojRoyI0047LZ5//vnSswAAAAaFAE7soYceitmz
#o> pngZ8djjz0Wv/jFL2LTpk1x4oknxoYNG0pPAwAA2OqaSw+gnMWLF/f79a233hojRoyIZcuWxWc+85lCqwAA
#o> pngAAaHK8D06ezsjIiIXXfdtfASAACArc8VYCIiore3Ny688MI45phj4qCDDhrwY7q7u6O7u7vv111dXdtq
#o> pngHgAAwBZzBZiIiJg9e3Y899xzcdddd73vx8yfPz/a2tr6Hu3t7dtwIQAAwJYRwMS5554bP/3pT+PBBx+M
#o> pngsWPHvu/HzZ07Nzo7O/seHR0d23AlAADAlnELdGK1Wi3OO++8uPfee2Pp0qUxYcKED/z4lpaWaGlp2Ubr
#o> pngAAAAti4BnNjs2bPjjjvuiPvvvz9aW1tj1apVERHR1tYWw4YNK7wOAABg63ILdGILFy6Mzs7OOO6442L0
#o> png6NF9j7vvvrv0NAAAgK3OFeDEarVa6QkAAADbjCvAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoC
#o> pngGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKzaUHsP1bOW33aGoZ
#o> pngWnpGn1GP9ZaeUHHKUx2lJwzoK393YekJFcN2qL/P36qOXUtPqNj/6bdLT6j4izlPl54woEenjS89oeKB
#o> pngZYtLT6j469/9WekJFc9cfGjpCRVnP1h/56iIiGnPdZWeULHv6DWlJ1Rc+doppSdUTLxvVekJFS+evKn0
#o> pnghAG9NvOg0hMqxvx8dekJ/Wzu6S49oe65AgwAAEAKAhgAAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABI
#o> pngQQADAACQggAGAAAgBQEMAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAA
#o> pngIAUBDAAAQAoCGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgA
#o> pngAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABIQQADAACQggAGAAAgBQEMAABACgIYAACAFAQwAAAAKQhg
#o> pngAAAAUmio1Wq10iPYPnV1dUVbW1vcuvyQ2LG1qfScPpc/N6P0hIqhPxteesKARtz3UukJFW+evE/pCRW9
#o> pngzaUXVO0z6/nSEyp+88bI0hMGdN2ke0pPqPgv//DV0hMqJv74jdITKobctK70hIqxO75desKAHvj1pNIT
#o> pngKk495FelJ1T8z4cml55QUWupv7fiLavr533dHxt37bLSEyoa28eUntDP5p7uWPLK9dHZ2RnDh9fn+8/S
#o> pngXAEGAAAgBQEMAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAA
#o> pngQAoCGAAAgBQEMAAAACkIYAAAAFIQwERExHe+851oaGiICy+8sPQUAACAQSGAiSeffDJ+9KMfxcEHH1x6
#o> pngCgAAwKARwMmtX78+zjrrrLj55ptjl112KT0HAABg0Ajg5GbPnh0nn3xyTJ069U9+bHd3d3R1dfV7AAAA
#o> pngbC+aSw+gnLvuuiuWL18eTz755If6+Pnz58eVV145yKsAAAAGhyvASXV0dMQFF1wQt99+ewwdOvRDvWbu
#o> png3LnR2dnZ9+jo6BjklQAAAFuPK8BJLVu2LNasWROf+tSn+p7r6emJhx9+OH74wx9Gd3d3NDU19XtNS0tL
#o> pngtLS0bOupAAAAW4UATuqEE06IX//61/2emzVrVuy///7xzW9+sxK/AAAA2zsBnFRra2scdNBB/Z7baaed
#o> pngYrfddqs8DwAA8HHge4ABAABIwRVg+ixdurT0BAAAgEHjCjAAAAApCGAAAABSEMAAAACkIIABAABIQQAD
#o> pngAACQggAGAAAgBQEMAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASKG59AC2f7cdvV80Nwwp
#o> pngPaNP0129pSdUNGwuvWBg/3rZXqUnVPzDjO+XnlBx0UXnlp5Q8euf7l96QsWxf/l06QkDuuq8L5WeULHv
#o> png0ytKT6hY++fjS0+o2PniYaUnVFz0P/6x9IQBtR/x+9ITKm5/eXLpCRV/P+O20hMqnvtDe+kJFQ/OPrr0
#o> pnghAH9zW8eLz2hYuYvzi49oZ/eP7wbcUHpFfXNFWAAAABSEMAAAACkIIABAABIQQADAACQggAGAAAgBQEM
#o> pngAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQE
#o> pngMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAAAAApCGAAAABS
#o> pngEMAAAACkIIABAABIQQADAACQggAGAAAgBQEMAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAA
#o> pngSEEAAwAAkEJDrVarlR7B9qmrqyva2tpi3mNTY+gndig9p8+xOz1fekLFhd+eXXrCgPb436tLT6h4Z+Ju
#o> pngpSdUrDm8fo7v93zl9MWlJ1SMG/Jm6QkDumnfvUpPqHjx+imlJ1QMXd1UekLFyGWbSk+oGLbirdITBrT6
#o> pnguD1KT6i4/7/+bekJFadd+Y3SEyp+f1hv6QkV+x/YUXrCgF6/d3zpCRWPf/P7pSf007WuN0bt1xGdnZ0x
#o> pngfPjw0nPqkivAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQEMAAAACkIYAAAAFIQwAAA
#o> pngAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAjix1157LT7/+c/HbrvtFsOGDYtJkybFU089VXoWAADA
#o> pngoGguPYAy3nrrrTjmmGPi+OOPj5/97Gexxx57xIsvvhi77LJL6WkAAACDQgAnde2110Z7e3v8+Mc/7ntu
#o> pngwoQJBRcBAAAMLrdAJ/WTn/wkJk+eHJ/73OdixIgRcdhhh8XNN99cehYAAMCgEcBJvfLKK7Fw4cKYOHFi
#o> png/PznP49zzjknzj///Fi0aNH7vqa7uzu6urr6PQAAALYXboFOqre3NyZPnhzXXHNNREQcdthh8dxzz8WN
#o> pngN94YM2fOHPA18+fPjyuvvHJbzgQAANhqXAFOavTo0XHAAQf0e+6Tn/xkvPrqq+/7mrlz50ZnZ2ffo6Oj
#o> pngY7BnAgAAbDWuACd1zDHHxPPPP9/vuRdeeCHGjRv3vq9paWmJlpaWwZ4GAAAwKFwBTuqiiy6Kxx57LK65
#o> png5pp46aWX4o477oibbropZs+eXXoaAADAoBDASR1xxBFx7733xp133hkHHXRQXHXVVbFgwYI466yzSk8D
#o> pngAAAYFG6BTuyUU06JU045pfQMAACAbcIVYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAK
#o> pngAhgAAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABIQQADAACQQnPpAWz/vrTzv8Tw1vr5Wsop511YekLF
#o> png7z9TKz1hQI2bR5aeUNG4qf7+rMZd90zpCRU3TTqm9ISKTa/vVHrCgPbb+TelJ1SM+mVD6QkVP5i/oPSE
#o> pngijnPnlt6QkVt2JDSEwbUuW/9nTvHNn+i9ISKndb0lJ5QMXSfN0tPqNjrE2tLTxjQhlfGlp5QMfO300pP
#o> png6GfTho0RcWvpGXWtfqoFAAAABpEABgAAIAUBDAAAQAoCGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggAEA
#o> pngAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABIQQADAACQggAG
#o> pngAAAgBQEMAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoC
#o> pngGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAAAAAp
#o> pngNJcewPZv+vfPi6aWoaVn9KnNfKv0hIqdSg94H2uGtZaeUHHA/N+VnlBx6lMrSk+ouOl7B5eeULHLC++W
#o> pngnjCgWk9v6QkVa/6i/v6s/vrFvyo9oWLoA8tKT6h4/vuTS08Y0J7/q6f0hIpPTTy99ISK3dd2l55QUdth
#o> pngU+kJFQ/dfXjpCQP6yQ3fLT2h4uszvlJ6Qj+be+rvGK83rgADAACQggAGAAAgBQEMAABACgIYAACAFAQw
#o> pngAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQEcFI9PT1x2WWXxYQJ
#o> pngE2LYsGGx9957x1VXXRW1Wq30NAAAgEHRXHoAZVx77bWxcOHCWLRoURx44IHx1FNPxaxZs6KtrS3OP//8
#o> png0vMAAAC2OgGc1C9/+cs49dRT4+STT46IiPHjx8edd94ZTzzxROFlAAAAg8Mt0EkdffTRsWTJknjhhRci
#o> pngIuKZZ56JRx55JKZPn154GQAAwOBwBTipSy+9NLq6umL//fePpqam6OnpiauvvjrOOuus931Nd3d3dHd3
#o> png9/26q6trW0wFAADYKlwBTuqee+6J22+/Pe64445Yvnx5LFq0KL73ve/FokWL3vc18+fPj7a2tr5He3v7
#o> pngNlwMAACwZQRwUt/4xjfi0ksvjTPOOCMmTZoUX/jCF+Kiiy6K+fPnv+9r5s6dG52dnX2Pjo6ObbgYAABg
#o> pngy7gFOql33nknGhv7f/2jqakpent73/c1LS0t0dLSMtjTAAAABoUATmrGjBlx9dVXx5577hkHHnhgPP30
#o> png03HdddfFl770pdLTAAAABoUATuoHP/hBXHbZZfH1r3891qxZE2PGjImvfe1rMW/evNLTAAAABoUATqq1
#o> pngtTUWLFgQCxYsKD0FAABgm/BDsAAAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAAAAApCGAAAABS
#o> pngEMAAAACkIIABAABIQQADAACQggAGAAAgBQEMAABACgIYAACAFJpLD2D717SxFk1RKz2jz6hLNpaeUPF3
#o> png//TfSk8Y0P/dvEvpCRVX/9MXS0+ouGVFS+kJFf/5op+XnlDxg8f/vPSEgW38ZOkFFXveWXpB1U7P9pSe
#o> pngUPE3Lz1ZekJFUzxResKAvtr++dITKnZ4YNfSEyreOLSh9ISKiS0rS0+ouOLrPyw9YUCvbG4rPaHioFuf
#o> pngLz2hn+71m+KfP116RX1zBRgAAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABIQQADAACQggAGAAAgBQEM
#o> pngAABACgIYAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQE
#o> pngMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwAAEAKAhgAAIAUBDAAAAApCGAAAABS
#o> pngEMAAAACkIIABAABIQQADAACQggAGAAAgBQEMAABACgIYAACAFAQwAAAAKQhgAAAAUmguPYDtV61Wi4iI
#o> pngno3vFl7S3+ae7tITKtav6y09YUDvbO4pPaFi86b6Op4iIno21N8x9e76zaUnVPT+of4+dxERsbH+vta7
#o> pngeVPpBVWbe+vvON9Qh+fOpqiVnjCgnnfq7/PXWGfvDyIiepsaSk+o2LRhY+kJFfX4dy8i4t1a/b1v6V5f
#o> pngXyf0jRv+/5733qdT1VDzp8NHtHLlymhvby89AwAA+CMdHR0xduzY0jPqkgDmI+vt7Y3XX389Wltbo6Fh
#o> pngy76i2tXVFe3t7dHR0RHDhw/fSguhvjjOycBxTgaOc+pVrVaLdevWxZgxY6Kxsf7ugKoHboHmI2tsbNzq
#o> pngX1kaPny4f0j42HOck4HjnAwc59Sjtra20hPqmi8LAAAAkIIABgAAIIWmK6644orSIyAioqmpKY477rho
#o> pngbnZnPh9fjnMycJyTgeMctk9+CBYAAAApuAUaAACAFAQwAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAA
#o> pngkIIABgAAIAUBDAAAQAoCGAAAgBQEMAAAACkIYAAAAFIQwAAAAKQggAEAAEhBAAMAAJCCAAYAACAFAQwA
#o> pngAEAKAhgAAIAUBDAAAAApCGAAAABSEMAAAACkIIABAABIQQADAACQggAGAAAgBQEMAABACgIYAACAFAQw
#o> pngAAAAKQhgAAAAUhDAAAAApCCAAQAASEEAAwAAkIIABgAAIAUBDAAAQAoCGAAAgBQEMAAAACkIYAAAAFIQ
#o> pngwAAAAKTw/wCYAs2ORAl7PgAAAABJRU5ErkJggg==
#o> png}}}

# vim: ft=python foldmethod=marker foldlevel=0
